pkg/apis/apps.kubermatic/v1/application_installation.go

```diff
@@ -55,6 +55,9 @@ type ApplicationInstallationSpec struct {
 	ApplicationRef ApplicationRef `json:"applicationRef"`
 
 	// Values describe overrides for manifest-rendering
+	// +kubebuilder:validation:Schemaless
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +kubebuilder:validation:Type=object
 	Values json.RawMessage `json:"values,omitempty"`
```

Comment:
A better way to solve this would be to use either of:
- "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1".JSON
- "k8s.io/apimachinery/pkg/runtime".RawExtension

They are perfect for our use case and built exactly for this scenario. With RawExtension you'll have to use `// +kubebuilder:pruning:PreserveUnknownFields` marker.